sdk/pinocchio/src/sysvars/epoch_rewards.rs

```diff
@@ -0,0 +1,50 @@
+use super::Sysvar;
+use crate::impl_sysvar_get;
+
+pub const HASH_BYTES: usize = 32;
+
+#[derive(Default, Debug, Clone)]
+pub struct Hash([u8; HASH_BYTES]);
```

Comment:
Should this also be `repr(C)` if it is used inside a `repr(C)` struct?

Reply:
@LStan I think it will be better to mimic the SDK type here – so avoid exposing the hash bytes:
```rust
[repr(transparent)]
pub struct Hash(pub(crate) [u8; HASH_BYTES]);
```
We eventually will use the SDK type so it will be easier to "swap" the type if they follow the same implementation.